test/lib/usd/pxrUsdPreviewSurface/testPxrUsdPreviewSurfaceExport.py

```diff
@@ -32,23 +31,27 @@ class testPxrUsdPreviewSurfaceExport(unittest.TestCase):
 
     @classmethod
     def setUpClass(cls):
-        standalone.initialize('usd')
+        testDir = os.path.join(os.path.abspath('.'),
```

Comment:
@ppt-adsk You are moving tests out of Pixar plugin for import/export - where should we be saving these temporary files? Should it go under temp folder (so follow AL example and usage of NamedTemporaryFile) or maybe you found a different solution?

Reply:
I believe I'm doing something similar to this, creating an output directory for the test if it is not already created, and emptying it out if it is.  This is in the build area.

Reply:
Why not leverage NamedTemporaryFile? (https://docs.python.org/3/library/tempfile.html)

Reply:
Why use that?  If I want to look at files after a test run, I know exactly where to look in the build area.  This is often very useful to diagnose test problems.

Reply:
The existing Pixar test run setup used temporary files, did not clean them up, and this left an increasing amount of useless files in the temporary area.

Reply:
Knowing where to look for them in a very good argument. I like it, but let's be a bit more specific about:
1. where in the build folder we should expect to find these temporary test files
2. rules about leaving and clearing the temporary test files folder

Reply:
As my implementation currently stands, the output files are placed in a per-test subdirectory of the build importExport directory, where all the import / export tests live, and where the .pyc files are written.  The subdirectory is called <theTest>Output, one per test.  The files are always left after the test run, and always cleaned at the start of the test run.  If you feel this deserves further conversation, we can do so outside this review.

Reply:
I would like us to align in how tests are dealing with it.

Reply:
>  The subdirectory is called Output, one per test. The files are always left after the test run, and always cleaned at the start of the test run.

This is great. @kxl-adsk Pierre's points makes a lot of sense to me. We can talk about this internally later so we are all aligned.